src/__test__/index.test.ts

```diff
@@ -33,4 +35,18 @@ describe('exports', () => {
 
 		expect(swup.version).not.toBeUndefined();
 	});
+
+	it('passes relative URL to ignoreVisit', () => {
+		let ignorableUrl = 'nothing';
+		const swup = new Swup({
+			ignoreVisit: (url, { el } = {}) => {
+				ignorableUrl = url;
+				return false;
+			}
```

Comment:
Can we make the `ignoreVisit` option a `jest.fn()` and check what is it being called with instead? Seems cleaner.